### ISSUE TITLE
Accessibility - JAWS - JAWS is not giving the current state information 

### ISSUE THREAD

**Issue by drotara** [microsoft/fluentui#16716]
### Environment Information

- **Package version(s)**: 7.153.2 + 
- **Browser and OS versions**: Edge - 88.0.705.56 

### Describe the issue:
JAWS does not read out the current state of the dropdown.

### Please provide a reproduction of the issue in a codepen:
https://codepen.io/adrotar/pen/BaQarzw

#### Actual behavior:
JAWS reads out the current selected data count and position

#### Expected behavior:
JAWS to read out the current state information as 'collapsed' or 'selected' as well as the selected data count and position.

### Documentation describing expected behavior
https://www.etsi.org/deliver/etsi_en/301500_301599/301549/02.01.02_60/en_301549v020102p.pdf
11.5.2.5 - Where the software provides a user interface it shall, by using the services as described in clause 11.5.2.3 [E.g. the Accessibility APIs, such as UIA on Windows], make the user interface elements’ role, state(s), boundary, name, and description programmatically determinable by assistive technologies.

**Comment by gouttierre** [comment]
@drotara  - thank you for filling this issue. I took a look at the codepen and also at our latest release. I see it does not mention the collapse state of if the item is selected. it cites for example: `Apple - One of seven - to a move to a an item use the arrow keys... "`

@khmakoto - Would you be able to confirm if this is a regression or an actual behavioral issue? Thanks
cc @smhigley
